gradle.properties

```diff
@@ -24,23 +24,23 @@ marshallingVersion=2.0.5.Final
 protobufVersion=3.20.3
 jacocoVersion=0.8.8
 
-stdlibIoVersion=1.5.0
-stdlibTimeVersion=2.3.0
-stdlibUrlVersion=2.3.0
+stdlibIoVersion=1.5.0-20230622-110500-8523dfe
```

Comment:
The standard library version should be the stable version rather than the timestamped version. This might be changed when resolving the conflicts. Cab we change them back to the stable versions by removing the timestamp part?

Reply:
Done